CHANGELOG.md

```diff
@@ -4,13 +4,21 @@ The Harmony Regridding Service follows semantic versioning. All notable changes
 to this project will be documented in this file. The format is based on [Keep a
 Changelog](http://keepachangelog.com/en/1.0.0/).
 
+## [unreleased] - 2025-09-17
+
+###
```

Comment:
Could this just be a `Changed`? I guess it doesn't quite fall under the normal use of the category, but that's the closest fit.